llama_stack/providers/adapters/inference/bedrock/bedrock.py

```diff
@@ -23,6 +23,10 @@
     "Llama3.1-8B-Instruct": "meta.llama3-1-8b-instruct-v1:0",
     "Llama3.1-70B-Instruct": "meta.llama3-1-70b-instruct-v1:0",
     "Llama3.1-405B-Instruct": "meta.llama3-1-405b-instruct-v1:0",
+    "Llama3.2-1B-Instruct": "us.meta.llama3-2-1b-instruct-v1:0",
```

Comment:
I dont think the model id has the `us.` prefix at the beginning. can you confirm?